src/Components/TopicArticles.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react"
+import { fetchArticles } from "../utils"
+import ListArticles from "./ListArticles"
+import { useParams } from "react-router-dom"
+
+export default function TopicArticles(){
+
+    const [articles, setArticles] = useState([])
+    const [page, setPage] = useState(1)
+    const [isLoading, setIsLoading] = useState(true)
+    const {topic} = useParams()
+
+    useEffect(()=>{
+        fetchArticles(page, 5, topic)
+        .then((result)=>{
+            setArticles(result);
+            setIsLoading(false);
+            
+        })
```

Comment:
Would be nice to handle errors here. At the moment the page crashes when something goes wrong.